discord/client.py

```diff
@@ -243,8 +243,8 @@ def __init__(
         )
 
         self._handlers: Dict[str, Callable] = {"ready": self._handle_ready}
-
         self._hooks: Dict[str, Callable] = {"before_identify": self._call_before_identify_hook}
+        self.events: Dict[str, Coro] = {}
```

Comment:
I’d suggest using a list or multidict here instead, to support multiple funcs on an event.
Also, I would suggest not removing trying to get that event Coro, it can and most likely will break a lot of subclasses.

Reply:
> Also, I would suggest not removing trying to get that event Coro, it can and most likely will break a lot of subclasses.
I have no idea what this means. Could you elaborate?

Reply:
People, on client or bot subclasses use a function on the subclass for some events, removing the code to validate that will break those functions